sandbox

```diff
@@ -698,6 +773,10 @@ EOF
       dump "$@"
       ;;
 
+    tail)
+      tail_f "$@"
```

Comment:
I found `dump` to be very helpful and added `tail`

Reply:
Should we deprecate `logs` ?

Reply:
nit: can you add a line for `tail` in the `help ()` printout?